cmd/kubeadm/app/cmd/phases/join/kubelet.go

```diff
@@ -93,13 +93,22 @@ func getKubeletStartJoinData(c workflow.RunData) (*kubeadmapi.JoinConfiguration,
 // runKubeletStartJoinPhase executes the kubelet TLS bootstrap process.
 // This process is executed by the kubelet and completes with the node joining the cluster
 // with a dedicates set of credentials as required by the node authorizer
-func runKubeletStartJoinPhase(c workflow.RunData) error {
+func runKubeletStartJoinPhase(c workflow.RunData) (err error) {
```

Comment:
named return appears required for the defer's error-handling

Reply:
`err` might be confusing because it is also redefined in many place in the code. 
What about `returnErr` or something similar?